src/game/phase/phase_inventory.c

```diff
@@ -203,6 +203,7 @@ static GAMEFLOW_OPTION Inv_Close(void)
 
     switch (g_InvChosen) {
     case O_PASSPORT_OPTION:
+        g_InvChosen = O_INVALID;
```

Comment:
This is the actual fix for the issue. The rest is just refactoring. I also reset it in `Game_Start` to be consistent? Lmk what you think.

Reply:
She still says no if she chooses a medpack, pistols, … etc.

Also please use a separate commit for `-1`, and yet another for removing the condition + `= 0` assignment.

Reply:
Yes on the commit separation.

But I'm confused on the first part. The issue is Lara saying "No" if you save or load in front of a key/puzzle hole. If you choose the wrong key item, she should say "No". Unless I am misunderstanding.

Reply:
Yes and no – if you're standing next to a keyhole, a gorilla is trying to murder you and you take a medkit, Lara understandably will complain about the situation but that's still not the right thing to do.

Reply:
Oh I never realized that. She also complains you select the pistol. I guess that's technically a different issue than this PR, but I will look into it.

Reply:
Sounds good. We also have the `Object_IsObjectType` and `g_EnemyObjects`. This should be tackled in #1268 though.

@walkawayy please rebase onto the newest develop and I'll approve. Thanks

Reply:
Well I might rework this PR to fix her saying No for other items too. Ideally, I would like to handle all the inventory stuff in the inventory function. That way we don't have this `g_InvChosen` thing that needs passed around and cleared at different spots in the code. I just need to think of a solution for these keys / puzzles. Lahm's idea might work. Also, I might need to set `Lara.is_moving` when it's confirmed the correct item is chosen so that the player can't input cancel due to the phaser change. It seems like tomb4 has a similar implementation to both of these ideas except for the puzzles/keys.